### DISCUSSION TITLE
no data is found here in elastic with Kibana

### DISCUSSION THREAD

**Discussion by Chunshan-Theta** [infiniflow/ragflow#1716] [Q&A] [answered]
Is it correct that no data is found here? I originally thought vector data would be stored here
![image](https://github.com/user-attachments/assets/ec8aecc7-b2cc-4bea-9958-50ccf5f1d90f)
![image](https://github.com/user-attachments/assets/a645d314-e1ed-42c7-8c84-937090bc6f44)

No data, Even if I already point to the index
![image](https://github.com/user-attachments/assets/d5e70b57-2caa-4902-acc4-dbf5a58cc021)

**Comment by KevinHuSh** [comment, marked as answer]
You could use ES dsl to query.

**Comment by KevinHuSh** [comment]
> Is it correct that no data is found here? I originally thought vector data would be stored here ![image](https://private-user-images.githubusercontent.com/13751822/352371786-ec8aecc7-b2cc-4bea-9958-50ccf5f1d90f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNjA3OTcsIm5iZiI6MTcyMjI2MDQ5NywicGF0aCI6Ii8xMzc1MTgyMi8zNTIzNzE3ODYtZWM4YWVjYzctYjJjYy00YmVhLTk5NTgtNTBjY2Y1ZjFkOTBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDEzNDEzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0ZjFiZDk1NDI4ZmE4MjkzMjUyMDg4ODY4NzAxYWVkZjBlNGFlMmRkYjJiYTk5NWNiOGQ4MWRhOGJhM2E0OTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.z1XNEuUFhmR7gM4qVRV5iathSba6Gp4bEAq222CuMI8) ![image](https://private-user-images.githubusercontent.com/13751822/352373351-a645d314-e1ed-42c7-8c84-937090bc6f44.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNjA3OTcsIm5iZiI6MTcyMjI2MDQ5NywicGF0aCI6Ii8xMzc1MTgyMi8zNTIzNzMzNTEtYTY0NWQzMTQtZTFlZC00MmM3LThjODQtOTM3MDkwYmM2ZjQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDEzNDEzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgyNjFhMDQyMjc3YjMyOWQ4ZDkxYmQ2Y2M5MTBjZGVlMWViOTk1NTg2ODY3MDdlOTZkYTlkMDcxYzM4M2Q5ZjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.66urUwc37LcKvZ6yOtBErCp5SGRHFA8-JBza2wiZjpY)
> 
> No data, Even if I already point to the index ![image](https://private-user-images.githubusercontent.com/13751822/352373757-d5e70b57-2caa-4902-acc4-dbf5a58cc021.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNjA3OTcsIm5iZiI6MTcyMjI2MDQ5NywicGF0aCI6Ii8xMzc1MTgyMi8zNTIzNzM3NTctZDVlNzBiNTctMmNhYS00OTAyLWFjYzQtZGJmNWE1OGNjMDIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDEzNDEzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmZTQwZTY1ZjgzNGNiZjdjODM4OTZmYjRlNWQzMTI3OWU2NzY3MTY0NDRiYTUyNTlmNmFjMzRiODA2MzFlZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tsqeo4Rk8zVbixJGk01xT12pkNYzL6ul_4Kq8QmN1t4)

The data do store in ES.
